### ISSUE TITLE
Malformed header in http request

### ISSUE THREAD

**Issue by shaneikennedy** [JupiterOne/jupiterone-api-client-python#20]
We're getting this error seemingly everywhere with the SDK v1.0.0
```
jupiterone.errors.JupiterOneApiError: 403:**** not a valid key=value pair (missing equal-sign) in Authorization header: 'Bearer ****'.
```

But as we use the SDK we are not touching the http headers

**Comment by SeaBlooms** [comment]
https://github.com/JupiterOne/jupiterone-api-client-python/blob/main/jupiterone/client.py#L56C14-L56C21
https://github.com/JupiterOne/jupiterone-api-client-python/blob/main/examples/examples.py#L9

Can you share how you're passing your token to the client creation? This was all tested and the headers look to be in the correct format.

**Comment by shaneikennedy** [comment]
`JupiterOneClient(account=J1_USER_NAME, token=J1_TOKEN, url=J1_HOST_URL)` this was working on version 0.2.1 yesterday

**Comment by shaneikennedy** [comment]
in this commit https://github.com/JupiterOne/jupiterone-api-client-python/commit/db821b76a878da5e8753c52c594edda4493ed5fe#diff-2dae7911702c04b3963617ac12304ef1645f20799da944ba65560c2e45732cc8 I see you change the default url, the one we pass in to the client generation is https://api.eu.jupiterone.io

does the api subdomain not work anymore?

**Comment by shaneikennedy** [comment]
> This was all tested and the headers look to be in the correct format.
Yeah when I searched around for this problem most of the solutions have nothing to do with malformed headers, I mostly made the issue title that because that's what the error message is so that people with similar problems with the j1 sdk could find this 👍

**Comment by SeaBlooms** [comment]
testing with "https://api.eu.jupiterone.io/" returns that same header error you raised.

current guidance is to update the URL to "https://graphql.eu.jupiterone.io" for EU region

up-to-date API reference docs here: https://docs.jupiterone.io/reference#introduction
